src/server/mod.rs

```diff
@@ -0,0 +1,158 @@
+pub mod bepinex;
```

Comment:
If this whole file becomes too large, could group by usage into different files and rexport public values here